timemachine/fe/utils.py

```diff
@@ -148,6 +149,16 @@ def get_romol_conf(mol):
     return guest_conf / 10  # from angstroms to nm
 
 
+def set_romol_conf(mol, new_coords):
+    """Sets coordinates of mol's 0th confomrmer"""
+    assert new_coords.shape[0] == mol.GetNumAtoms()
+    # convert from nm to angstroms
+    angstrom_coords = new_coords * 10
+    conf = mol.GetConformer(0)
+    for i, (x, y, z) in enumerate(angstrom_coords):
+        conf.SetAtomPosition(i, Point3D(float(x), float(y), float(z)))
```

Comment:
nit: is there a downside to this slightly simpler version?
```suggestion
    for i, pos in enumerate(angstrom_coords):
        conf.SetAtomPosition(i, pos)
```

Reply:
The API requires a Point3D object and each value must be in a `float` 

https://www.rdkit.org/docs/source/rdkit.Chem.rdchem.html#rdkit.Chem.rdchem.Conformer.SetAtomPosition

Unless you are indicating a larger change?

Reply:
No larger change; the Python API seems to also accept iterables of objects that can be converted to float (the docs are not clear on this, but do seem to show a variant that takes `AtomPairsParameters` 😕 ). In any case, there are many examples of passing e.g. lists and tuples as the second argument in the [rdkit repo](https://github.com/rdkit/rdkit/search?q=SetAtomPosition), so either is probably fine?